docs/app-deployment-v2.md

```diff
@@ -0,0 +1,160 @@
+
+# Application Configuration
+
+>Note: Depending on the [Migration Status](/README.md#Migration status) and the environment you are adding config, you should also do [v1 setup](app-deployment.md)
+
+The section covers how to configure a new application in this repository to deploy to various environments. We use [kustomize](https://github.com/kubernetes-sigs/kustomize) for templating/patching manifests in this repo. 
+
+**It is important to follow below discussed naming convention, file and folder names should match application helm release name**
+
+Note: Some of these scripts need you to [install yq](https://mikefarah.gitbook.io/yq/)
```

Comment:
english isn't right on above, this could work of changing it to a 'Prerequisites'
```suggestion
Note: You need to [install yq](https://mikefarah.gitbook.io/yq/) for these scripts
```